baselines/dev/bootstrap.sh

```diff
@@ -2,22 +2,14 @@
 set -e
 cd "$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/../
 
-version=${1:-3.7.12}
-
-# Destroy and recreate the venv
-./dev/venv-delete.sh $version
-./dev/venv-create.sh $version
-
 # Remove caches
 ./dev/rm-caches.sh
 
-# Remove poetry.lock file
-rm -f poetry.lock
-
 # Upgrade/install spcific versions of `pip`, `setuptools`, and `poetry`
 python -m pip install -U pip==22.2
 python -m pip install -U setuptools==63.2.0
 python -m pip install -U poetry==1.1.14
 
 # Use `poetry` to install project dependencies
-python -m poetry install
+python -m poetry install \
+  --extras "simulation"
```

Comment:
```suggestion
python -m poetry install
```

Reply:
The `simulation` extra is only available in the `flwr` package, which is a direct dependency of `flwr_baselines` (`flwr_baselines` doesn't have extra dependencies).

Reply:
Oops, fixed it